aviary/docs/getting_started/onboarding_level2_test.ipynb

```diff
@@ -0,0 +1,1260 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cd250ed3",
+   "metadata": {},
+   "source": [
+    "# Level 2\n",
+    "\n",
+    "We have discussed [Level 1](onboarding_level1) in great detail. As we have seen, Level 1 is relatively simple.\n",
+    "\n",
+    "In Level 2, we see more flexibility and additional choices. You will be exposed to some simple Python code and see how Aviary establishes a problem from beginning to end. Each step is built upon the capabilities of [Level 3](onboarding_level3). Users will be led to Level 3 in a natural way."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1e1804c6",
+   "metadata": {
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Testing Cell\n",
+    "import inspect\n",
+    "import aviary.api as av\n",
+    "from aviary.api import Dynamic, Mission\n",
+    "from aviary.interface.methods_for_level1 import run_aviary\n",
+    "from aviary.utils.doctape import glue_variable, get_variable_name\n",
+    "\n",
+    "glue_variable('interface/methods_for_level1.py', md_code=True, display=False)\n",
+    "glue_variable('methods_for_level1.py', md_code=True, display=False)\n",
+    "glue_variable('run_aviary()', md_code=True, display=False)\n",
+    "AviaryValues = av.AviaryValues\n",
+    "glue_variable('AviaryValues', f'{AviaryValues=}'.split('=')[0], md_code=True, display=False)\n",
+    "\n",
+    "# glue all argument of function run_aviary()\n",
+    "sigs = inspect.signature(run_aviary)\n",
+    "parameters = sigs.parameters\n",
+    "for name, param in parameters.items():\n",
+    "    glue_variable(name, md_code=True, display=False)\n",
+    "    # print(f\"Name: {name}, Default: {param.default}, Kind: {param.kind}\")\n",
+    "\n",
+    "glue_variable('SIZING', av.ProblemType.SIZING.name, md_code=True, display=False)\n",
+    "glue_variable('ALTERNATE', av.ProblemType.ALTERNATE.name, md_code=True, display=False)\n",
+    "glue_variable('FALLOUT', av.ProblemType.FALLOUT.name, md_code=True, display=False)\n",
```

Comment:
you could use glue_keys here instead of glueing each one individually:
`glue_keys(dict(av.ProblemType.__members__), md_code=True)`